app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = Payment.where(user_id: current_user.id)
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, 'f')
```

Comment:
This reads strange. What is `'f`'? (That's a rhetorical question, I know it means unpaid tickets, but just by glancing at this line, I cannot figure that out).

Something like this would read much more nicely:

```ruby
current_users.tickets.by_conference(@conference).unpaid.total_price
```